src/serialize.h

```diff
@@ -768,7 +768,7 @@ void Unserialize_impl(Stream& is, prevector<N, T>& v, const V&)
         nMid += 5000000 / sizeof(T);
         if (nMid > nSize)
             nMid = nSize;
-        v.resize(nMid);
+        v.resize_uninitialized(nMid);
         for (; i < nMid; i++)
```

Comment:
The btc issue has it ++i
`for (; i < nMid; ++i)`

Reply:
both were okay. done anyway.